src/components/Common/PortfolioItem.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+class PortfolioItem extends Component{
```

Comment:
If components has no state, there's no need for a class component. Give it a try and change this to a functional component.